template_snippets/cloudconfig/net_bridge.yaml

```diff
@@ -0,0 +1,39 @@
+{{define "net_bridge"}}
+  - name: 00-{{.Host.ConnectedNIC}}.network
+    runtime: false
+    content: |
+      [Match]
+      Name={{.Host.ConnectedNIC}}
+
+      [Network]
+      Address={{.Host.InternalAddr}}/22
+      Gateway={{.ClusterNetwork.Router}}
+      {{ range $server := .ClusterNetwork.DNS }}DNS={{ $server }}
+      {{ end }}
+  - name: 00-br0.netdev
+    runtime: true
+    content: |
+      [NetDev]
+      Name=br0
+      Kind=bridge
+
+  - name: 30-br0.network
+    runtime: true
+    content: |
+      [Match]
+      Name=br0
+
+      [Network]
+      Address=192.168.10.1/24
+      DHCPServer=yes
+      DNS=8.8.8.8
+      IPMasquerade=yes
+  - name: 30-nodhcp.network
```

Comment:
if you want to be really sure that this is only the default for all interfaces that haven't been configured it would be good to call it `99-nodhcp-default.network`

similar to the `99-default.link` that is shipped with the system usually.